source/user-manual/reference/ossec-conf/active-response.rst

```diff
@@ -118,16 +118,19 @@ Defines a minimum severity level required for the command to be executed.
 rules_group
 ^^^^^^^^^^^
 
-Defines the rule group that a rule must belong to one for the command to be executed.
+Defines the rule group that a rule must belong to for the execution of the command.
 
-+--------------------+---------------------------------------------------------------------------------------------+
-| **Default value**  | n/a                                                                                         |
-+--------------------+---------------------------------------------------------------------------------------------+
-| **Allowed values** | Any rule group is allowed. Multiple groups should be separated with a pipe character (“|”). |
-+--------------------+---------------------------------------------------------------------------------------------+
++--------------------+------------------------------------------------------------------------------------------+
+| **Default value**  | n/a                                                                                      |
++--------------------+------------------------------------------------------------------------------------------+
+| **Allowed values** | Any group string. For multiple groups, separate the strings with a pipe character ``|``. |
++--------------------+------------------------------------------------------------------------------------------+
 
 .. note::
-	All groups must end with a comma.
+	
+   To avoid partial matches, add a comma at the end of the group string. For example, ``<rules_group>group_a,|group_b,|group_c,</rules_group>`` Also, check that the rule group in your rules definitions ends with a comma as well. This is usually the case in the Wazuh default ruleset. For example, ``<group>group_b,</group>``.
```

Comment:
```suggestion
   To avoid partial matches, add a comma at the end of the group string. For example, ``<rules_group>group_a,|group_b,|group_c,</rules_group>`` Also, check that the rule group in your rule definitions ends with a comma as well. This is usually the case in the Wazuh default ruleset. For example, ``<group>group_b,</group>``.
```